cpp/src/gandiva/function_registry_common.h

```diff
@@ -175,6 +175,12 @@ typedef std::unordered_map<const FunctionSignature*, const NativeFunction*, KeyH
   NativeFunction(#NAME, std::vector<std::string> ALIASES, DataTypeVector{TYPE()}, \
                  date64(), kResultNullIfNull, ARROW_STRINGIFY(NAME##_from_##TYPE))
 
+#define NEXT_DAY_SAFE_NULL_IF_NULL(NAME, ALIASES, TYPE)    \
```

Comment:
Where are you calling this macro to ensure that function is registered?

Reply:
Also, It would be good if you add an integrated test to check that you registered the function correctly.